blueprints/incredible-squaring/deploy.ts

```diff
@@ -1,12 +1,12 @@
 import * as fs from "node:fs/promises";
```

Comment:
When it comes to the `gadget-cli`, we need to be automating this type of file as much as possible. Things like:
- Defining constants / variables to testnet/chain should be streamlined
- Deploying the contracts with the proper constructor arguments, our defaults can be empty (TODOs or other comments indicating what might be changed as if it's a coding exercise)
- Calling all jobs, since we are able to generate json, we should be able to iterate over all jobs and generate the TS scripts for calling things. This can likely be done in both TS and Rust. We want to streamline the devex / testing / deployment infra.